lib/hanami/events/adapter/redis.rb

```diff
@@ -15,11 +15,12 @@ def initialize(params)
           @logger = params[:logger]
           @subscribers = []
           @thread_spawned = false
+          @stream = params[:stream] || STREAM_NAME
         end
 
         def broadcast(event_name, payload)
           @redis.with do |conn|
-            conn.lpush(STREAM_NAME, {
+            conn.lpush(@stream, {
```

Comment:
WDYT, maybe is better to create getter `stream` instead instance variable? I mean:
```ruby
def stream
  @stream || DEFAULT_STREAM
end
```